content/language/c-plus-plus/containerize.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Containerize a C++ application
+keywords: C++, containerize, initialize
+description: Learn how to containerize a C++ application.
+aliases:
+  - /language/c-plus-plus/build-images/
+  - /language/c-plus-plus/run-containers/
+---
+
+## Prerequisites
+
+* You have a [git client](https://git-scm.com/downloads). The examples in this section use a command-line based git client, but you can use any client.
+
+## Overview
+
+This section walks you through containerizing and running a C++ application.
+
+## Get the sample application
+
+Clone the sample application to use with this guide. Open a terminal, change directory to a directory that you want to work in, and run the following command to clone the repository:
+
+```console
+$ git clone https://github.com/Pradumnasaraf/c-plus-plus-docker.git
```

Comment:
Can we fork your repo to a Docker-controlled repo and update this?

Reply:
Yes, go ahead, Craig.